### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN048.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#97] [General]
# https://blog.3place.cn/3place/lmyn/LMYN048.html

https://blog.3place.cn/3place/lmyn/LMYN048.html

**Comment by picklecai** [comment]
他俩照亮了这个地方，
有如日月照耀天宇〔381〕。

有人觉得你来了他就幸福。
有人觉得你的光辉照耀了他的地方。这不就是“蓬荜生辉”？

**Comment by picklecai** [comment]
他垂涎仙人乔达摩的妻子阿科里耶，于是让月神帮忙，半夜化成公鸡，高声啼叫。让乔达摩仙人误以为天亮，急急忙忙跑去晨祷，因陀罗趁虚而入，强行霸占了阿科里耶。乔达摩仙人知道后，勃然大怒，但无奈实力不如因陀罗，于是便诅咒因陀罗，让他身上打上一千个女性私处的记号。再后来，这一千个记号变成了因陀罗的一千只眼睛，因陀罗也因此得名千眼大神，可是他的名声却一落千丈。

怪不得感觉这个故事见过。

**Comment by picklecai** [comment]
这两位王子，愿你们有福！
勇敢可以比得上神仙，
步伐像是大象和狮子，
两位英雄老虎、公牛一般。 （1.47.2）

他俩的大眼睛像荷叶，
随身带着刀剑、箭壶和弓；
样子长得像那阿湿波，
他俩正是在青春妙龄。 （1.47.3）

他俩像是天上的神仙，
偶然降落到了人间。

神仙最勇敢。
大象和狮子的步伐，那就是“静若初女，动若脱兔”。
老虎凶猛，公牛力大。
眼睛像荷叶。印度人的审美也觉得眼睛大好看、有力量。

天神降落，就类似于“这个婆娘不是人，九天仙女下凡尘”。

**Reply by picklecai** [reply]
突然觉得，这段和这个很像：
手如柔荑，
肤如凝脂，
领如蝤蛴，
齿如瓠犀，
螓首蛾眉。
巧笑倩兮，
美目盼兮。

**Comment by picklecai** [comment]
这牟尼具备苦行的神力，
神仙和檀那婆难以近身。
他刚用圣河中的水洗净，
像火焰一般光辉动人。
他手里拿着木柴俱舍草，

俱舍草，到底有啥用？天天拿在手上。

**Comment by picklecai** [comment]
所有的那些仙人们，
看到了遮那竭〔386〕美丽的城。
“好哇，好哇！”他们赞叹，
一起向弥提罗城致敬。 （1.47.10）

是不是悉多要出场了？

**Comment by picklecai** [comment]
听完了罗摩说的话，
擅长辞令的大牟尼，
这非常光辉的众友，
就来回答他的问题： （1.47.13）

众友确实擅长辞令。

**Comment by picklecai** [comment]
他看到大牟尼乔达摩，
从外面一直走进了门。
这牟尼具备苦行的神力，
神仙和檀那婆难以近身。
他刚用圣河中的水洗净，
像火焰一般光辉动人。
他手里拿着木柴俱舍草，
这一位牟尼中的师尊。 （1.47.23）

俱舍草为什么神奇？下面这个故事提到了。

[佛经传奇故事：两姐妹同时嫁给仙人，“你们想要什么样的儿子？”](https://baijiahao.baidu.com/s?id=1766789877579366022&wfr=spider&for=pc )

返回蛇岛，伽拉多对黑蛇们说：“我已经拿回了长生不死甘露，药放在俱舍草上，现在你们应该遵守协定，放了我母亲吧？”
黑蛇们非常高兴，他们连连点头，释放了伽拉多的母亲。接着，他们洗漱一番，想好好品尝一下甘露，等他们来到俱舍草边时，因陀罗早就拿走了甘露。
没办法，黑蛇们只好舔放过药瓶子的俱舍草，结果舌头都分了叉，而俱舍草也因此变成一种圣草。

**Comment by picklecai** [comment]
他看到他干了坏事，
怒不可遏把话来讲： （1.47.25）
’你这个坏东西，
变成了我的样子，
干了不应干的勾当，
从此将不能有子嗣。’ （1.47.26）
高贵尊严的乔达摩，
怒气冲冲地这样说。
千眼大神的睾丸，
立刻往地上掉落。 （1.47.27）

不能有子嗣，就用如此直白的方式……
